cross/libvpx/Makefile

```diff
@@ -0,0 +1,37 @@
+PKG_NAME = libvpx
+PKG_VERS = 1.3.0
+PKG_EXT = tar.bz2
+PKG_DIST_NAME = $(PKG_NAME)-v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://webm.googlecode.com/files
+PKG_DIR = $(PKG_NAME)-v$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = http://www.webmproject.org/code/
+COMMENT  =
+
+CONFIGURE_ARGS = --disable-sse --disable-sse2 --disable-sse3 --disable-ssse3 --disable-sse4_1 --enable-shared --enable-vp8 --enable-vp9 --disable-examples --disable-docs --enable-postproc --enable-vp9-postproc --enable-pic --disable-altivec --disable-unit_tests --prefix=$(INSTALL_PREFIX)
+
+include ../../mk/spksrc.cross-cc.mk
+
+ENV = LD= CPP= NM= CC= RANLIB= CXX= AR= STRIP= OBJDUM= CFLAGS="$(CFLAGS)" CPPFLAGS= LDFLAGS="$(LDFLAGS)" CXXFLAGS= READELF= AS= OBJDUMP= DESTDIR=$(INSTALL_DIR) CROSS=$(TC_PATH)$(TC_PREFIX)
```

Comment:
Oh, right. I just noticed we discussed this before...guess I forgot after all this time ;) Carry on.